phing/tasks/tests.xml

```diff
@@ -63,6 +63,10 @@
               passthru="true"
               spawn="true"
               checkreturn="true" />
+        <echo message="Waiting 10 seconds for ${behat.server-url} to become available."/>
```

Comment:
I think that the behat.server-url isn't actually what you'd like to test here. The port is wrong.